lib/modules/dosomething/dosomething_signup/dosomething_signup.install

```diff
@@ -625,3 +631,22 @@ function dosomething_signup_update_7022() {
     db_add_unique_key($table, 'uid-nid-run_nid', ['uid', 'nid', 'run_nid']);
   }
 }
+
+/**
+ * Adds campaign run nid column to the dosomething_signup table.
+ */
+function dosomething_signup_update_7023() {
```

Comment:
I'm not sure where this came from! Could it have been from a rebase of dev since this PR hasn't been touched for awhile?